src/PowerShellEditorServices/Services/DebugAdapter/Handlers/DebuggerActionHandlers.cs

```diff
@@ -4,59 +4,68 @@
 using System;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Extensions.Logging;
 using Microsoft.PowerShell.EditorServices.Services;
 using OmniSharp.Extensions.DebugAdapter.Protocol.Requests;
 using OmniSharp.Extensions.JsonRpc;
 
 namespace Microsoft.PowerShell.EditorServices.Handlers
 {
-    // TODO: Inherit from ABCs instead of satisfying interfaces.
-    internal class DebuggerActionHandlers : IContinueHandler, INextHandler, IPauseHandler, IStepInHandler, IStepOutHandler
+    internal class ContinueHandler : ContinueHandlerBase
     {
-        private readonly ILogger _logger;
         private readonly DebugService _debugService;
-
-        public DebuggerActionHandlers(
-            ILoggerFactory loggerFactory,
-            DebugService debugService)
-        {
-            _logger = loggerFactory.CreateLogger<ContinueHandlerBase>();
-            _debugService = debugService;
-        }
-
-        public Task<ContinueResponse> Handle(ContinueArguments request, CancellationToken cancellationToken)
+        public ContinueHandler(DebugService debugService) { _debugService = debugService; }
+        public override Task<ContinueResponse> Handle(ContinueArguments request, CancellationToken cancellationToken)
         {
             _debugService.Continue();
             return Task.FromResult(new ContinueResponse());
         }
+    }
 
-        public Task<NextResponse> Handle(NextArguments request, CancellationToken cancellationToken)
+    internal class NextHandler : NextHandlerBase
+    {
+        private readonly DebugService _debugService;
```

Comment:
Usually yes but it's a big file of verbose duplicates because of OmniSharp. (Still prefer the abstract base classes to the interfaces though, they were introduced in the v0.19 update and _for the most part_ make things way simpler, this was like the one exception but I think it's better to be consistent.)